index.js

```diff
@@ -204,7 +213,16 @@ function appInit (opts) {
   // enable catching <href a=""></href> links
   // enable HTML5 history API
   if (opts.href !== false) pushLocationSub(href)
-  if (opts.history !== false) pushLocationSub(history)
+  if (opts.history !== false) {
+    pushLocationSub(history)
+  // otherwise enable hash history
+  } else {
```

Comment:
perhaps have `opts.hash === true` as the trigger? can imagine there's situations where neither is desirable to enable